content/root/platform-customization.textile

```diff
@@ -57,6 +57,10 @@ Note that this does not prevent customers outside of your chosen region from con
 
 If you are interested in applying processing or storage constraints to regions other than Europe or the US, then please speak to your customer success manager.
 
+Once your account is enabled and configured to support regional routing and storage, to emulate regional restriction at the application level, you would need to specify the required environment: @eu@ or environment: @us@ in the Ably client options ( "Rest SDK":/docs/api/rest-sdk#client-options and "Realtime SDK":/docs/api/realtime-sdk#client-options ) when you are initiating the connection, from both the publisher's and subscriber's side.
```

Comment:
Suggestion: Maybe bring the 'subject' to the beginning of this sentence. Is 'implement' a better word to use than 'emulate'?

Maybe 👇 
To implement regional restrictions at the application level, you must specify the required environment after configuring your account for regional routing and storage.